README.md

```diff
@@ -22,10 +35,38 @@ Make sure `$GOPATH/bin` is in your `$PATH`.
       -token string
         	github access token
 
-You'll need a Github access token, which you can generate
-[here](https://github.com/settings/tokens). Dave only needs `repo` scope to
-work.
-
 All flags have an environment variable that sets the default, so setting
 `DAVE_REPO` also changes the repo Dave looks at. Flags override environment
 variables.
+
+## Contributing
+
+See the [CONTRIBUTING.md](CONTRIBUTING.md) file on how to contribute to this project.
+
+## Contributors
+
+See the [CONTRIBUTORS.md](CONTRIBUTORS.md) file for a list of contributors to the project.
+
+## Roadmap
```

Comment:
Instead of having Roadmap, including Changelog, in README.md, I'd rather see this in a separate file to prevent README.md from becoming too long. (And for Dave, there's no Roadmap)